astropy/modeling/separable.py

```diff
@@ -142,8 +141,9 @@ def _arith_oper(left, right):
     -------
     result : ndarray
         Result from this operation.
-    """
-    # models have the same number of inputs and outputs
+    """  # noqa: D202
+
+    # Models have the same number of inputs and outputs
```

Comment:
The comment should be the docstring on the function below it, then the `D202` can be dropped.

Reply:
Done!